lib/src/__examples__/contract-with-config.ts

```diff
@@ -0,0 +1,131 @@
+import {
+  api,
+  body,
+  config,
+  defaultResponse,
+  endpoint,
+  Float,
+  headers,
+  pathParams,
+  queryParams,
+  request,
+  response,
+  securityHeader,
+  String,
+  test
+} from "@airtasker/spot";
+import "./contract-endpoint";
+import { Address, ErrorBody, UserBody } from "./models";
+
+/** This is the company API. It does cool things */
+@api({ name: "company-api" })
+@config({
```

Comment:
Have you considered making `@api` accept a `config` property, instead of creating a new `@config` decorator?

Reply:
Yes, when I chatted with @lfportal, we considered a couple of approaches, including the one you're describing. We ended up choosing `@config` because it looked nicer than the other approaches.

Also, we discarded `config` inside `@api` because, from the spot docs, `@api` is meant to have meta information only.

Does it make sense?

Reply:
You could argue that is is meta information? I'm happy either way, so feel free to resolve this thread :)